tests/lib/rules/multiple-assertions-checker.js

```diff
@@ -0,0 +1,93 @@
+"use strict";
+
+const rule = require("../../../lib/rules/multiple-assertions-checker");
+const RuleTester = require("../../../node_modules/eslint/lib/testers/rule-tester");
+
+const ruleTester = new RuleTester();
+ruleTester.run("multiple-assertions-checker", rule, {
+	valid: [
+`describe('functionUnderTest():', function() {
+  it('should do things under test', function() {
+    const result = { a: true, b: false };
+    expect(result.a).to.equal(true);
+  });
+});`,
+`describe('functionUnderTest():', function() {
+  it('should do things under test', function() {
+    const result = { a: true, b: false };
+    expect(result.a, 'result.a test failed').to.equal(true);
+    expect(result.b, 'result.b test failed').to.equal(true);
+  });
+});`,
+`describe('when a property is selected', function() {
+  beforeEach(function() {
+    this.context = buildContext();
+    angular.mock.inject((SelectPropertyUiData, $q) => {
+      sinon.stub(SelectPropertyUiData, 'findProperty').returns($q.resolve([ property ]));
+    });
+    this.context.component.selectProperty = property.propertyId;
+  });
+
+  it('should enable the "Save & Next" button', function () {
+    expect(this.context.component.nextStepButton).to.not.be.disabled;
+  });
+
+  it('should not show the input box', function () {
+    expect(this.context.component.selectProperty.element).to.not.exist;
+  });
+});`
+	],
```

Comment:
This code's a bit difficult to read, partly because of the indentation. I'd recommend breaking it up a bit, and indenting as follows:

```
const invalid = [{
  errors: [{ message: 'whatevs'  }],
  code: `
   describe('functionUnderTest():', function() {
     it('should do things under test', function() {
       const result = { a: true, b: false };
       expect(result.a).to.equal(true);
       expect(result.b).to.equal(true);
      });
    });`
}];

const valid = [{ /* etc */ }];
new RuleTester.run("multiple-assertions-checker", rule, { valid, invalid });
  
```